gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/V2PipelineTemplatesController.java

```diff
@@ -68,9 +67,9 @@ public V2PipelineTemplatesController(
     this.objectMapper = objectMapper;
   }
 
-  @ApiOperation(value = "(ALPHA) List pipeline templates.", response = List.class)
+  @ApiOperation(value = "(ALPHA) List pipeline templates.", response = HashMap.class)
```

Comment:
The returns a `Map` but the API operation is annotated as a `HashMap`...should this be `Map.class` here?

Reply:
:+1: thanks!